stream-chat-android-state/src/main/java/io/getstream/chat/android/state/event/handler/internal/EventHandlerSequential.kt

```diff
@@ -268,49 +268,79 @@ internal class EventHandlerSequential(
     private suspend fun updateGlobalState(batchEvent: BatchEvent) {
         logger.v { "[updateGlobalState] batchId: ${batchEvent.id}, batchEvent.size: ${batchEvent.size}" }
 
+        var me = clientState.user.value
+        var totalUnreadCount = mutableGlobalState.totalUnreadCount.value
+        var channelUnreadCount = mutableGlobalState.channelUnreadCount.value
+
+        val modifyValues = { event: HasUnreadCounts ->
+            me =
+                me?.copy(
+                    totalUnreadCount = event.totalUnreadCount,
+                    unreadChannels = event.unreadChannels,
+                )
```

Comment:
It looks like we can delete this, since we've moved `clientState.user` updates to `ChatClient` 👍